app/grandchallenge/workstations/migrations/0003_group_data_migration.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2.4 on 2019-08-09 08:15
+
+from django.db import migrations
+
+
+def delete_workstation_groups(apps, schema_editor):
+    Workstation = apps.get_model("workstations", "Workstation")
+
+    for ws in Workstation.objects.all():
+        ws.editors_group.delete()
+        ws.users_group.delete()
+
+        ws.editors_group = None
+        ws.users_group = None
+
+        ws.save()
+
+
+def create_workstation_groups(apps, schema_editor):
+    from grandchallenge.workstations.models import Workstation
+
+    for ws in Workstation.objects.all():
+        ws.save(force_group_creation=True)
```

Comment:
It's used further down for assigning permissions too. The other option is duplicating the group creation and permissions code within the migration itself, or, by using a `pre_save` and `post_save` signal (something that I *really* want to avoid design-wise). I guess, we do need to archive this piece of code as what was going on at the point in time of the data migration, so I will see if it moves ok.